packages/grpc-js/src/call-stream.ts

```diff
@@ -129,7 +125,8 @@ export class Http2CallStream extends Duplex implements Call {
     private readonly channel: ChannelImplementation,
     private readonly options: CallStreamOptions,
     filterStackFactory: FilterStackFactory,
-    private readonly channelCallCredentials: CallCredentials
+    private readonly channelCallCredentials: CallCredentials,
+    private readonly callNumber: number
```

Comment:
Sort of design nit here: why does it have to be an actual number? Wouldn't a string be just as effective, and more flexible / extensible, if needs be?

Reply:
I think it depends on how this ID might get used in the future. If it's just used in logs, then it would be fine to use a string or number or whatever. But I think there are use cases where it makes more sense to have a number. Either way, I don't think it really matters. This is internal to the library, and can be changed in the future if there is a reason to do so.